docs/tr/docs/python-types.md

```diff
@@ -144,58 +140,94 @@ Yalnızca `str` değil, tüm standart Python tiplerinin bildirebilirsiniz.
 
-Bu tür tip belirteçlerini desteklemek için özel olarak mevcuttur.
+Tüm dokümanlarda Python'un her sürümüyle uyumlu örnekler bulabilirsiniz (eğer bir farklılık oluştuysa).
 
-#### `List`
+Örneğin "**Python 3.6+**" Python 3.6 veya üzeri (3.7, 3.8, 3.9, 3.10, vb. dahil) ile uyumlu olduğu anlamına gelir. Ve "**Python 3.9+**" Python 3.9 veya üzeri (3.10 vb. dahil) ile uyumlu olduğu anlamına gelir.
+
+**Python'un en son sürümlerini** kullanayorsanız, en son sürüm için oluşturulan örnekleri kullanın. Bunlar **en iyi ve en basit sözdizimine** sahip olacaktır, örneğin, "**Python 3.10+**".
+
+#### `List` (Liste)
 
 Örneğin `str` değerlerden oluşan bir `list` tanımlayalım.
 
-From `typing`, import `List` (büyük harf olan `L` ile):
+=== "Python 3.9+"
 
-```Python hl_lines="1"
-{!../../../docs_src/python_types/tutorial006.py!}
-```
+    Declare the variable, with the same colon (`:`) syntax.
 
-Değişkenin tipini yine iki nokta üstüste (`:`) ile belirleyin.
+    As the type, put `list`.
 
-tip olarak `List` kullanın.
+    As the list is a type that contains some internal types, you put them in square brackets:
 
-Liste, bazı dahili tipleri içeren bir tür olduğundan, bunları köşeli parantez içine alırsınız:
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial006_py39.py!}
+    ```
 
-```Python hl_lines="4"
-{!../../../docs_src/python_types/tutorial006.py!}
-```
+=== "Python 3.6+"
+
+    From `typing`, import `List` (büyük harf olan `L` ile):
+
+    ``` Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial006.py!}
+    ```
+
+    Değişkenin tipini yine iki nokta üstüste (`:`) ile belirleyin ve tip olarak `List` kullanın.
+
+    Liste, bazı dahili tipleri içeren bir tür olduğundan, bunları köşeli parantez içine alırsınız:
+
+    ```Python hl_lines="4"
+    {!> ../../../docs_src/python_types/tutorial006.py!}
 
 !!! ipucu
     Köşeli parantez içindeki bu dahili tiplere "tip parametreleri" denir.
 
-    Bu durumda `str`, `List`e iletilen tür parametresidir.
+    Bu durumda `str`, `List`e iletilen tür parametresidir (veya Python 3.9+ için `list`)
 
 Bunun anlamı şudur: "`items` değişkeni bir `list`tir ve bu listedeki öğelerin her biri bir `str`dir".
 
-Bunu yaparak, düzenleyicinizin listedeki öğeleri işlerken bile destek sağlamasını sağlayabilirsiniz:
+!!! tip
+    Python 3.9 veya daha üstünü kullanıyorsanız, `List`i `typing`den import etmek zorunda değilsiniz, bunun yerine aynı normal `list` türünü kullanabilirsiniz.
+
+Bunu yaparak, editörünüz listedeki öğeleri işlerken bile destek sağlayabilir:
 
 <img src="/img/python-types/image05.png">
 
-Tip belirteçleri olmadan, bunu başarmak neredeyse imkansızdır.
+Tipler olmadan bunu başarmak neredeyse imkansızdır.
 
-`item` değişkeninin `items` listesindeki öğelerden biri olduğuna dikkat edin.
+Değişken öğenin `list` öğelerindeki `items`dan biri olduğuna dikkat edin.
```

Comment:
Hocam otomatik çeviri mi kullandık?

``Notice that the variable `item` is one of the elements in the list `items`.``

``Şuna dikkat edin: `item` değişkeni, `items` listesinde bulunan elemanlardan birisi.``

`item` değişkenine değinilmemiş.